app/actions.go

```diff
@@ -89,7 +91,9 @@ var insertApp = action.Action{
 			return nil, err
 		}
 		defer conn.Close()
-		app.Quota = quota.Unlimited
+		if app.Quota == (appTypes.Quota{}) {
```

Comment:
Shouldn't you just set `app.Quota.Limit = quota.Unlimited`?

Reply:
Yes. Changing to reference `UnlimitedQuota`, both in `app` and in `quota`. Now to create a generic unlimited quota just copy from that struct.